index.html

```diff
@@ -19,10 +20,188 @@ <h1 class="textdeco1 text1width">José Fernández-Caminero</h1>
+        <div class="blueBox">
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="button1 width65 buttonMargin1">See this project
+                <img src="resources/Union.png" alt="Cooler Arrow Pointing Right">
+                <img src="resources/Union2.svg" alt="Cool (Blue) Arrow Pointing Right">
+            </button>
+        </div>
+    
+        <img class="fullWidth" src="resources/Rectangle 21(4).png" alt="Laptop and other objects">
+        <div class="blueBox">
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="button1 width65 buttonMargin1">See this project
+                <img src="resources/Union.png" alt="An Even Cooler Arrow Pointing Right">
+                <img src="resources/Union2.svg" alt="Cool (Blue) Arrow Pointing Right">
+            </button>
+        </div>
+
+        <div>
+            <img class="fullWidth" src="resources/Rectangle 21(5).png" alt="Another laptop and other objects">
+            <div class="blueBox">
+                <h3 class="projectName">Project name goes here</h3>
+
+                <ul class="greenul">
+                    <li class="greenli">HTML/CSS</li>
+                    <li class="greenli">Ruby on Rails</li>
+                    <li class="greenli">JavaScript</li>
+                </ul>
+
+                <button class="button1 width65 buttonMargin1">See this project
+                    <img src="resources/Union.png" alt="Cool (White) Arrow Pointing Right">
+                    <img src="resources/Union2.svg" alt="Cool (Blue) Arrow Pointing Right">
+                </button>
+            </div>
+        </div>
+
+        <img class="fullWidth" src="resources/Rectangle 21(6).png" alt="Picture of laptop and other objects">
+        <div class="blueBox">
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="button1 width65 buttonMargin1">See this project
+                <img src="resources/Union.png" alt="Yet An Even Cooler Arrow Pointing Right">
+                <img src="resources/Union2.svg" alt="Cool (Blue) Arrow Pointing Right">
+            </button>
+        </div>
+    </section>
+
+    <header>
+        <div>
+            <h1 class="textdeco2">About</h1>
```

Comment:
For semantic best practices, search engine optimization (SEO) and nice user experience (UX), don't you think we should have only one h1 tag in an HTML file? 🤔  You can check this [docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) for more info 👍 

I will suggest you use other heading <h2 - h6>, according to their order of importance 🙂